code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -97,6 +99,8 @@
 	var/mob/living/carbon/carbon_user = user
 	if(istype(carbon_user))
 		carbon_user.update_spacesuit_hud_icon("0")
+	if(isplasmaman(user))
+		slowdown = 0.5
```

Comment:
Same thing here

Reply:
so you're fine with plasmamen being slightly faster when wearing a spacesuit than everyone else?

Reply:
yeah it's not a big deal